LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentRequest.java

```diff
@@ -45,6 +45,22 @@ public class LocalPaymentRequest {
     private String phone;
     private boolean shippingAddressRequired;
     private String surname;
+    private final boolean hasUserLocationConsent;
+
+    /**
+     * Deprecated. Use {@link LocalPaymentRequest#LocalPaymentRequest(boolean)} instead.
+     **/
+    @Deprecated
+    public LocalPaymentRequest() {
+        hasUserLocationConsent = false;
+    }
+
+    /**
```

Comment:
Should these exist in this PR?

Reply:
We don't have the approved copy yet. I feel that having them merged into the feature branch will save time once we have the copy. But happy to remove them if folks feel differently!

Reply:
Ah gotcha, yeah if we are waiting on copy to address the docstrings in a future PR before merging the feature into main that's 👍 to me.